<launch>
    <node pkg="sick_gmod_communication" type="communication.py" name="communication" output="screen" />
</launch>